tidb-cloud/tidb-cloud-billing.md

```diff
@@ -21,6 +21,14 @@ See [{{{ .starter }}} Pricing Details](https://www.pingcap.com/tidb-cloud-starte
 
 For {{{ .essential }}}, you are charged based on the number of provisioned Request Capacity Units (RCUs), **not** on the actual usage by your application. See [{{{ .essential }}} Pricing Details](https://www.pingcap.com/tidb-cloud-essential-pricing-details/).
 
+<CustomContent plan="premium">
+### Pricing for Premium
+
+For TiDB Cloud Premium, you are billed based on the number of provisioned Request Capacity Units (RCUs) and the storage you actually use, rather than the underlying backend nodes or provisioned disk size. As Premium is still in Private Preview , you can [contact our sales](https://www.pingcap.com/contact-us/) for pricing.
```

Comment:
```suggestion
For TiDB Cloud Premium, you are billed based on the number of provisioned Request Capacity Units (RCUs) and the storage you actually use, rather than the underlying backend nodes or provisioned disk size. As Premium is currently in private preview, you can [contact our sales](https://www.pingcap.com/contact-us/) for pricing details.
```